tsconfig.json

```diff
@@ -18,8 +19,19 @@
       "node",
       "jasmine",
       "fontfaceobserver"
-    ]
+    ],
+    "baseUrl": ".",
+    "paths": {
+      "@blackbaud/skyux-builder/*": [
+        "./node_modules/@blackbaud/skyux-builder/*",
```

Comment:
Added this to tell TS about the webpack aliases.
https://www.typescriptlang.org/docs/handbook/module-resolution.html#path-mapping